Docs.md

```diff
@@ -63,6 +63,17 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
 If you are embedding deno in a Rust program, see
 [the rust docs](https://deno.land/rustdoc/deno/index.html).
 
+## Browser compatibility
+
+Not all Deno programs will run in the browser, but a subset of them can. For
+example, a TypeScript program will not run in the browser without transpiling.
+Therefore it's important to explicitly and complete define what compatibility
```

Comment:
Did you mean to say `completely define` here?

Reply:
Fixed.